builder/azure/common/template/template.go

```diff
@@ -52,19 +60,19 @@ type OSDiskUnion struct {
 	Caching      compute.CachingTypes              `json:"caching,omitempty"`
 	CreateOption compute.DiskCreateOptionTypes     `json:"createOption,omitempty"`
 	DiskSizeGB   *int32                            `json:"diskSizeGB,omitempty"`
-	ManagedDisk  *compute.ManagedDiskParameters    `json:"managedDisk,omitempty"`
+	ManagedDisk  *ManagedDisk                      `json:"managedDisk,omitempty"`
```

Comment:
This is because of the API version change, correct?

Reply:
for ManagedDiskParameters, I was having trouble setting the DiskEncryptionID so I just defined more of the object in our template, it's the same json that the compute package uses